### ISSUE TITLE
Menu Bar: Display highest temperature

### ISSUE THREAD

**Issue by jamesst20** [crystalidea/macs-fan-control#385]
Hi,

I would like to be able to display the highest temperature on the menu bar. It's not very useful to have let's say 'CPU Core 1' if it is the Core 3 that is getting hot. Also, if it's the GPU that is running hot, it would be better to have the hottest GPU sensor displayed instead.

Thank you

**Comment by tbenko73** [comment]
I agree, I also suggested a feature to allow controlling the fans based on highest temperature. It would make more sense...

**Comment by traisjames** [comment]
I also agree with using the highest temperature for setting the fan speed.
